openapi.yml

```diff
@@ -794,7 +835,6 @@ components:
       additionalProperties: false
       allOf:
         - $ref: "#/components/schemas/DescriptiveStructuredValue"
```

Comment:
I realized a $ref for DescriptiveValueLanguage should be added to Event after all, to correspond to originInfo attributes.